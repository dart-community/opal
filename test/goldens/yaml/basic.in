name: Dart
version: 3
stable: true
type: "programming language"
tags: ["object-oriented", "functional"]
