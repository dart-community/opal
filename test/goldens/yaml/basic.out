name: Dart
^^^^
source-text-data-yaml-literal-string-unquoted
    ^
    source-text-data-yaml-punctuation-separator
     ^
     source-text-data-yaml-whitespace
      ^^^^
      source-text-data-yaml-literal-string-unquoted
version: 3
^^^^^^^
source-text-data-yaml-literal-string-unquoted
       ^
       source-text-data-yaml-punctuation-separator
        ^
        source-text-data-yaml-whitespace
         ^
         source-text-data-yaml-literal-string-unquoted
stable: true
^^^^^^
source-text-data-yaml-literal-string-unquoted
      ^
      source-text-data-yaml-punctuation-separator
       ^
       source-text-data-yaml-whitespace
        ^^^^
        source-text-data-yaml-literal-string-unquoted
type: "programming language"
^^^^
source-text-data-yaml-literal-string-unquoted
    ^
    source-text-data-yaml-punctuation-separator
     ^
     source-text-data-yaml-whitespace
      ^
      source-text-data-yaml-literal-string-quoted-double-literal-string-quoted-double-begin
       ^^^^^^^^^^^^^^^^^^^^
       source-text-data-yaml-literal-string-quoted-double-literal-string-unquoted
                           ^
                           source-text-data-yaml-literal-string-quoted-double-literal-string-quoted-double-end
tags: ["object-oriented", "functional"]
^^^^
source-text-data-yaml-literal-string-unquoted
    ^
    source-text-data-yaml-punctuation-separator
     ^
     source-text-data-yaml-whitespace
      ^
      source-text-data-yaml-punctuation
       ^
       source-text-data-yaml-literal-string-quoted-double-literal-string-quoted-double-begin
        ^^^^^^^^^^^^^^^
        source-text-data-yaml-literal-string-quoted-double-literal-string-unquoted
                       ^
                       source-text-data-yaml-literal-string-quoted-double-literal-string-quoted-double-end
                        ^
                        source-text-data-yaml-punctuation-separator
                         ^
                         source-text-data-yaml-whitespace
                          ^
                          source-text-data-yaml-literal-string-quoted-double-literal-string-quoted-double-begin
                           ^^^^^^^^^^
                           source-text-data-yaml-literal-string-quoted-double-literal-string-unquoted
                                     ^
                                     source-text-data-yaml-literal-string-quoted-double-literal-string-quoted-double-end
                                      ^
                                      source-text-data-yaml-punctuation
